src/rgw/rgw_perf_counters.cc

```diff
@@ -124,6 +124,11 @@ std::shared_ptr<PerfCounters> create_rgw_op_counters(const std::string& name, Ce
   add_rgw_op_counters(&pcb);
   std::shared_ptr<PerfCounters> new_counters(pcb.create_perf_counters());
   cct->get_perfcounters_collection()->add(new_counters.get());
+  new_counters->time_filtered = cct->_conf.get_val<bool>("rgw_op_counters_time_filtered");
+  if (new_counters->time_filtered) {
+    new_counters->time_alive = std::chrono::seconds(cct->_conf.get_val<uint64_t>("rgw_op_counters_dump_expiration"));
```

Comment:
do we really need two separate knobs? for rgw_op_counters_dump_expiration, could 0 mean disabled?

Reply:
My thinking behind the two knobs is that it follows the same patter we have in having two knobs  for the user and bucket counters cache. We could just have the cache size be 0 and as the check for not having them set at all, but we remove that boolean as well.

Random question. If the user set a negative value for the time config parameter would that be represented as a (probably high) positive integer for the number of seconds?